{"name": "Успешний логин пользователя с корректными данными", "status": "passed", "description": "Проверка успешной авторизации пользователя с корректными данными", "start": 1703365594963, "stop": 1703365596244, "uuid": "b61dcecb-afe4-4c4e-9f7e-edba392c5835", "historyId": "9675e9b2121f0eb0b435d3d7b6be803b", "testCaseId": "9675e9b2121f0eb0b435d3d7b6be803b", "fullName": "tests.test_login_user.TestLoginUser#test_login_user_with_correct_data_success", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_login_user"}, {"name": "subSuite", "value": "TestLoginUser"}, {"name": "host", "value": "DESKTOP-C4ER3HN"}, {"name": "thread", "value": "19912-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_login_user"}]}